spark-3.3/clickhouse-spark/src/main/scala/org/apache/spark/sql/clickhouse/ClickHouseSQLConf.scala

```diff
@@ -209,4 +209,13 @@ object ClickHouseSQLConf {
       .stringConf
       .transform(_.toLowerCase)
       .createWithDefault("binary")
+
+  val READ_WITH_SETTINGS: ConfigEntry[String] =
+    buildConf("spark.clickhouse.read.withSettings")
+      .doc("Query-level settings when reading from ClickHouse. e.g. `final=1, max_execution_time=5`")
```

Comment:
from the spark perspective, it's a session-level configuration

```
SET spark.clickhouse.read.settings=final=1;
SELECT * FROM ch.db.t1; -- affected
SELECT * FROM ch.db.t2; -- affected too
```

Reply:
agree. will update the doc

Reply:
updated